### DISCUSSION TITLE
Unable to access prisma studio

### DISCUSSION THREAD

**Discussion by AnirudhMemani** [prisma/prisma#25530] [Q&A]
### Bug description

I was trying to run `npx prisma studio` on my PostgreSQL database from neon.tech and it shows the below pop-up.

![Screenshot 2024-10-25 054031](https://github.com/user-attachments/assets/f05c6bf9-6d09-46e9-a89f-2012d5b3c009)

### How to reproduce

1. cd to directory your prisma folder is in
2. Run `npx prisma studio`

### Expected behavior

Shows the tables

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: Windows
- Database: PostgreSQL

### Prisma Version

```
Prisma version 5.20.0
```

**Comment by ludralph** [comment]
Hi @AnirudhMemani 👋 

If you're still running into issues, please provide a few more details by filling in the original issue with information on the relevant logs from the pop-up.

Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
